help/view-and-browse-images.md

```diff
@@ -1,36 +1,83 @@
 # View and browse images
 
 When someone pastes or attaches an image to a message, Zulip shows a small
-preview. Click on the image preview to open the **image viewer**.
+preview. You can open the image viewer to see an image in more detail or
+download it.
 
-In the image viewer, you can:
+## Using the image viewer
 
-* Zoom in and out of the image
+{start_tabs}
 
-* Click and drag the image
+{tab|desktop-web}
 
-* **Reset zoom** so that the image is recentered and its original size
+1. Click on the image preview to open the **image viewer**.
 
-* **Open** the image in a new browser tab
+1. Interact with the image.
 
-* **Download** the image
+    * Zoom in and out of the image
 
-* Browse other images in the current view. For example, if you're in a
-  stream view, the image browser will show all the images from that stream. If
-  you do a [search](/help/search-for-messages), the image browser will show
-  all images in messages that matched that search.
+    * Click and drag the image
 
-Exit the image viewer by clicking anywhere outside the image.
+    * **Reset zoom** so that the image is recentered and back to its original size
+
+    * **Open** the image in a new browser tab if you are using the Zulip web app
+
+    * **Download** the image
+
+1. Click anywhere outside the image to close the image viewer once you are done.
 
 !!! keyboard_tip ""
 
     Use <kbd>V</kbd> to **open** the image viewer. Use <kbd>Shift</kbd> +
     <kbd>Z</kbd> and <kbd>Z</kbd> to zoom in and out of the image. Use
     <kbd>V</kbd> or <kbd>Esc</kbd> to **close** the image viewer.
 
+{tab|mobile}
+
+1. Tap the image preview or file name to open the **image viewer**.
+
+1. Interact with the image.
+
+    * Zoom in and out of the image
+
+    * Pan the image if you are zoomed in
+
+    * Tap the **ellipsis** (<i class="zulip-icon zulip-icon-more-vertical-spread"></i>)
+    in the bottom right corner of the app to download the image, share it to other
+    apps, or share a link to the image
+
+1. Tap the **X** in the upper right corner of the app to close the image viewer
+   once you are done.
+
+{end_tabs}
+
+## Using the image browser
+
+In the Zulip desktop or web app, you can browse the images in the current view.
+For example, if you're in a stream view, the image browser will show all the
```

Comment:
Let's use a conversation view as the example here, rather than a stream view. We're generally moving in the direction of reducing the prominence of interleaved stream views.